README.md

```diff
@@ -1 +1,113 @@
-## WidgetBot documentation
+[assets]: https://github.com/widgetbot-io/documentation/tree/master/docs/assets
+[embed]: https://github.com/widgetbot-io/documentation/tree/master/docs/embed
+[guides]: https://github.com/widgetbot-io/documentation/tree/master/docs/guides
+[self-hosted]: https://github.com/widgetbot-io/documentation/tree/master/docs/self-hosted]
+[tutorial]: https://github.com/widgetbot-io/documentation/tree/master/docs/tutorial
+[crate]: https://github.com/widgetbot-io/documentation/tree/master/docs/embed/crate
+[html-embed]: https://github.com/widgetbot-io/documentation/tree/master/docs/embed/html-embed
+[react-embed]: https://github.com/widgetbot-io/documentation/tree/master/docs/embed/react-embed
+
+# WidgetBot documentation
+This repository contains the documentation, that can be found at https://docs.widgetbot.io/  
+Any contributions and improvements of the documentation are welcome.
+
+# Content
+
+## [assets]
+Contains files like images and general css/js stuff for defining the style of the site.  
+Please don't touch those.
+
+## [embed]
+Contains documentation about the [crate], the [html-embed] and the [react-embed].
+
+## [guides]
+General guides about fixing issues (currently how to switch to a cluster).
+
+## [self-hosted]
+Information about how to self-host the WidgetBot.
+
+## [tutorial]
+Tutorials.
+
+# Styling
+The used sites are saved as markdown-files (`.md`) and use markdown and some basic html, like iframes.  
+But there is also some own additional styling used in order to provide additional stuff.
+
+## Boxes
```

Comment:
The boxes section is not needed, instead it can just be a link to https://squidfunk.github.io/mkdocs-material/extensions/admonition.